portal-web/docroot/html/taglib/ui/input_localized/page.jsp

```diff
@@ -291,6 +303,14 @@
 			Liferay.InputLocalized.register(
 				'<%= namespace + id + HtmlUtil.getAUICompatibleId(fieldSuffix) %>',
 				{
+					<c:if test="<%= Validator.isNotNull(activeLocales) %>">
+						activeLocales: <%= JSONFactoryUtil.createJSONArray(activeLocales) %>,
+					</c:if>
+
+					<c:if test="<%= adminMode %>">
```

Comment:
mmm why? Can't you pass adminMode directly?

Reply:
mmmm good question, maybe this is a leftover, let me check, nice catch!

Reply:
Done! https://github.com/liferay-frontend/liferay-portal/pull/1328/commits/4b6a9e7c2519e668f250f8c3b2879dd452d8453d

Reply:
👏